contracts/src/periphery/cert/interfaces/IEigenDACertVerifierRouter.sol

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {IEigenDACertVerifierBase} from "src/periphery/cert/interfaces/IEigenDACertVerifierBase.sol";
+
+interface IEigenDACertVerifierRouter is IEigenDACertVerifierBase {
+    /// @notice Returns the address for a cert verifier at a given ABN.
+    function getCertVerifierAt(uint32 abn) external view returns (address);
```

Comment:
I think the input might be better as `rbn` since it's what would be passed in here